build.gradle

```diff
@@ -194,6 +189,37 @@ project('segmentstore:storage:impl') {
     }
 }
 
+
+project('segmentstore:storage:bindings') {
```

Comment:
THis appears to be dumping all of the dependencies into one target. Ideally we should have separate bindings for FS, HDFS, ExtS3, etc.